Userland/DevTools/HackStudio/main.cpp

```diff
@@ -76,9 +78,12 @@ int main(int argc, char** argv)
     };
 
     window->show();
-
     s_hack_studio_widget->update_actions();
 
+    if (mode_coredump) {
+        s_hack_studio_widget->open_coredump(argument_absolute_path);
+    }
```

Comment:
no curly braces needed for single-line if body

```suggestion
    if (mode_coredump)
        s_hack_studio_widget->open_coredump(argument_absolute_path);
```